guides/v2.2/cloud/project/project-conf-files_services.md

```diff
@@ -6,96 +6,139 @@ functional_areas:
+Renaming a service in the `services.yaml` file **permanently removes** the following:
 
 -  The existing service before creating a new service with the new name you specify.
 -  All existing data for the service is removed. We strongly recommend you [snapshot your environment]({{ page.baseurl }}/cloud/project/project-webint-snap.html) before you change the name of an existing service.
 
-### `type` {#cloud-yaml-services-type}
+### `type`
 
-The `type` of your service in the format: `type:version`
+The `type` value specifies the version of the service. For example:
 
-We support and deploy the following services for you:
+```yaml
+mysql:
+    type: mysql:10.2
+```
 
--  [`mysql`]({{ page.baseurl }}/cloud/project/project-conf-files_services-mysql.html)
--  [`redis`]({{ page.baseurl }}/cloud/project/project-conf-files_services-redis.html)
--  [`elasticsearch`]({{ page.baseurl }}/cloud/project/project-conf-files_services-elastic.html)
--  [`rabbitmq`]({{ page.baseurl }}/cloud/project/project-conf-files_services-rabbit.html)
+### `disk`
+
+The `disk` value specifies the size of the persistent disk storage (in MB) to allocate to the service. Services that use persistent storage, such as MySQL, must provide a disk value. Services that use memory instead of persistent storage, such as Redis, do not require a disk value.
+
+```yaml
+mysql:
+    type: mysql:10.2
+    disk: 2048
+```
+
+The current default storage amount per project is 5GB, or 5120MB. You can distribute this amount between your application and each of its services.
+
+## Service relationships
 
-### `disk` {#cloud-yaml-services-disk}
+{{site.data.var.ece}} uses the [`$MAGENTO_CLOUD_RELATIONSHIPS`]({{page.baseurl}}/cloud/env/environment-vars_cloud.html) environment variable to retrieve environment-related relationships. For services to be available to an application in your project, you must specify [*relationships*]({{ page.baseurl }}/cloud/project/project-conf-files_magento-app.html#relationships) between applications and services in the `.magento.app.yaml` file.
+
+#### To verify relationships in local environment:
+
+```bash
+magento-cloud relationships
+```
+
+**Abbreviated sample response**:
+
+```terminal
+redis:
+    -
+...
+        type: 'redis:3.2'
+        port: 6379
+elasticsearch:
+    -
+...
+        type: 'elasticsearch:6.5'
+        port: 9200
+database:
+    -
+...
+        type: 'mysql:10.0'
+        port: 3306
+
+```
+{:.no-copy}
 
-Specifies the size of the persistent disk storage (in MB) allocated to the service.
+#### To verify relationships in remote environments:
 
-For example, the current default storage amount per project is 5GB, or 5120MB. You can distribute this amount between your application and each of its services.
+1.  Use SSH to log in to the remote environment.
 
-## Using the services
+1.  Create `pretty-print` of all relationships for services and configuration data for that environment.
```

Comment:
After you create the pretty-print, what's next?    Review and test the connections?